bundles/org.openhab.binding.tibber/src/main/java/org/openhab/binding/tibber/internal/TibberBindingConstants.java

```diff
@@ -34,55 +34,89 @@ public class TibberBindingConstants {
+    public static final String CHANNEL_AVERAGE = "average";
+
+    // live channels
+    public static final String CHANNEL_CONSUMPTION = "consumption";
+    public static final String CHANNEL_MIN_COSNUMPTION = "minimum-consumption";
+    public static final String CHANNEL_PEAK_CONSUMPTION = "peak-consumption";
+    public static final String CHANNEL_PRODUCTION = "production";
+    public static final String CHANNEL_MIN_PRODUCTION = "minimum-production";
+    public static final String CHANNEL_PEAK_PRODUCTION = "peak-production";
+    public static final String CHANNEL_VOLTAGE_1 = "voltage1";
+    public static final String CHANNEL_VOLTAGE_2 = "voltage2";
+    public static final String CHANNEL_VOLTAGE_3 = "voltage3";
+    public static final String CHANNEL_CURRENT_1 = "current1";
+    public static final String CHANNEL_CURRENT_2 = "current2";
+    public static final String CHANNEL_CURRENT_3 = "current3";
+
+    // statistics channels
+    public static final String CHANNEL_TOTAL_CONSUMPTION = "total-consumption";
+    public static final String CHANNEL_DAILY_CONSUMPTION = "daily-consumption";
+    public static final String CHANNEL_DAILY_COST = "daily-cost";
+    public static final String CHANNEL_LAST_HOUR_CONSUMPTION = "last-hour-consumption";
+    public static final String CHANNEL_TOTAL_PRODUCTION = "total-production";
+    public static final String CHANNEL_DAILY_PRODUCTION = "daily-production";
+    public static final String CHANNEL_LAST_HOUR_PRODUCTION = "last-hour-production";
+
+    public static final String CURRENCY_QUERY_RESOURCE_PATH = "/graphql/currency.graphql";
+    public static final String PRICE_QUERY_RESOURCE_PATH = "/graphql/prices.graphql";
+    public static final String REALTIME_QUERY_RESOURCE_PATH = "/graphql/realtime.graphql";
+    public static final String WEBSOCKET_SUBSCRIPTION_RESOURCE_PATH = "/graphql/websocket.graphql";
+
+    public static final String SCHEDULE_CONTAINER = "{\"size\":%s, \"cost\":%s, \"schedule\":%s}";
+    public static final String QUERY_CONTAINER = "{\"query\":\"%s\"}";
+    public static final String WEBSOCKET_URL_QUERY = "{\"query\":\"{viewer {websocketSubscriptionUrl }}\"}";
+    public static final String CONNECT_MESSAGE = "{\"type\":\"connection_init\", \"payload\":{\"token\":\"%s\"}}";
+    public static final String DISCONNECT_MESSAGE = "{\"type\":\"connection_terminate\",\"payload\":null}";
+    public static final String SUBSCRIPTION_MESSAGE = "{\"id\":\"1\",\"type\":\"subscribe\",\"payload\":{\"variables\":{},\"extensions\":{},\"operationName\":null,\"query\":\"subscription { liveMeasurement(homeId:\\\"%s\\\") { timestamp power lastMeterConsumption lastMeterProduction accumulatedConsumption accumulatedConsumptionLastHour accumulatedCost currency minPower averagePower maxPower voltagePhase1 voltagePhase2 voltagePhase3 currentL1 currentL2 currentL3 powerProduction accumulatedProduction accumulatedProductionLastHour minPowerProduction maxPowerProduction }}\"}}";
```

Comment:
For future refactoring it would be a step forward to create proper DTO's for these

Reply:
See https://github.com/openhab/openhab-addons/pull/18726#discussion_r2194377365